man/man8/zpool.8

```diff
@@ -35,7 +35,9 @@
 .Nd configure ZFS storage pools
 .Sh SYNOPSIS
 .Nm
-.Fl ?
+.Fl ?V
+.Nm
+.Cm version
```

Comment:
I always end up fighting with mandoc too, but I believe this is what we want.  This puts the `zpool -V --version` alias first, and the `zpool version` subcommand with the others in alphabetical order.

```diff
diff --git a/man/man8/zpool.8 b/man/man8/zpool.8
index 7ba8a5bae9..410f34afee 100644
--- a/man/man8/zpool.8
+++ b/man/man8/zpool.8
@@ -35,7 +35,7 @@
 .Nd configure ZFS storage pools
 .Sh SYNOPSIS
 .Nm
-.Fl ?
+.Fl ?V
 .Nm
 .Cm add
 .Op Fl fgLnP
@@ -210,6 +210,8 @@
 .Cm upgrade
 .Op Fl V Ar version
 .Fl a Ns | Ns Ar pool Ns ...
+.Nm
+.Cm version
 .Sh DESCRIPTION
 The
 .Nm
@@ -909,6 +911,13 @@ The following subcommands are supported:
 Displays a help message.
 .It Xo
 .Nm
+.Fl V, -version
+.Xc
+An alias for the
+.Nm zpool Cm version
+subcommand.
+.It Xo
+.Nm
 .Cm add
 .Op Fl fgLnP
 .Oo Fl o Ar property Ns = Ns Ar value Oc
@@ -2441,6 +2450,14 @@ flag is specified, no features will be enabled on the pool.
 This option can only be used to increase the version number up to the last
 supported legacy version number.
 .El
+.It Xo
+.Nm
+.Cm version
+.Xc
+Displays the software version of the
+.Nm
+userland utility and the zfs kernel module.
+.El
 .El
 .Sh EXIT STATUS
 The following exit values are returned:
```

You're going to need to do something similar for `zfs(8)`.

Reply:
done